aws/README.md

```diff
@@ -42,6 +42,10 @@ I generally download and install these tools by hand (verifying checksums when a
 1. Install [Bundler](http://bundler.io/) **==1.16.2**, probably via `gem install bundler -v 1.16.2`.
 1. Install [rake](https://github.com/ruby/rake) **==12.3.0**, probably via `gem install rake -v 12.3.0`.
 1. Install [jq](https://stedolan.github.io/jq/) **==1.5**.
+1. If on macOS version prior to High Sierra (10.13), install curl with openssl
+   (`brew install curl --with-openssl` and add to your path by adding following line
+   to your shell rc `export PATH="/usr/local/opt/curl/bin:$PATH" >> ~/zshrc`).
```

Comment:
I think that a user who benefits from this example is more likely to be using bash than zsh ;). Can you change it to `>> ~/.bashrc`?

Reply:
The known case of macOS curl-openssl issues is just one of possible combinations that disables SNI in insecure mode. Situation with custom `curl` dependency can be avoided by using 3rd party cert-checking service instead of direct call, something like:
```
#  curl -X POST -d "host=preferences.dev-cindyli.gpii.net" https://www.digicert.com/api/check-host.php 2>&1 | grep -Eo "Issuer =([^<]*)"
Issuer = Fake LE Intermediate X1
```
Can't say if it any is better, since it introduces another dependency on web service that can go down, change its API, etc...

Reply:
Relying on 3rd party website is not desired, it would be difficult to impossible to debug, as we have no control nor visibility over it (is connectivity between me & the website down, connectivity between website & gpii site, did they just change something or is the website just down?...). In general, introducing 3rd party dependencies into your tests is an anti-pattern.

@natarajaya can you please elaborate on the other combinations that disable SNI? This was the one mentioned in the ticket.

Reply:
@mrtyler updated example in docs to bash